scss/layout/_grid.scss

```diff
@@ -119,14 +130,32 @@
     }
   }
 
-  .slab .grid {
-    .grid__item:nth-last-child(-n+2) {
-      border-bottom: none;
-      margin-bottom: 0;
+  .slab {
+    .grid {
+      h3 {
+        line-height: 1.5;
```

Comment:
Feels a little weird to have this property nested in `.slab > .grid`. Should it be a more generalized class or applied to all h3s?